src/Apps/Sell/SellFlowContext.tsx

```diff
@@ -169,10 +169,22 @@ export const SellFlowContextProvider: React.FC<SellFlowContextProviderProps> = (
   useEffect(() => {
     const newStep = steps[index]
 
-    if (isNewSubmission || !newStep) return
+    // If we are navigating away from the Sell flow, we don't want to redirect
+    const isSellFlowRoute = match.location.pathname.includes(
+      "/sell/submissions"
+    )
```

Comment:
Not checking the pathname leads to a bug that prevents navigating away from the Sell flow. This occurs because the effect is called and navigates to the "next" step in certain cases after navigating to a new URL. 🤷